src/sync/polling/updaters/splitChangesUpdater.ts

```diff
@@ -139,8 +140,12 @@ export function splitChangesUpdaterFactory(
             splits.setChangeNumber(splitChanges.till),
             splits.addSplits(mutation.added),
             splits.removeSplits(mutation.removed),
-            segments.registerSegments(mutation.segments)
+            segments.registerSegments(mutation.segments),
           ]).then(() => {
+            if (telemetryTracker) {
```

Comment:
Emma, here we are incrementing the value for all split and segment changes (coming from streaming events or not). 
I think the correct places are:
- `SplitsUpdateWorker` for `.trackUpdatesFromSSE(SPLITS);`
- `MySegmentsUpdateWorker` for `.trackUpdatesFromSSE(MY_SEGMENT);`

Please Emma, check also with the team where are they incrementing the value, in what component, for a second opinion.